scripts/qa_crowbarsetup.sh

```diff
@@ -4208,6 +4208,59 @@ EOF
     exit $ret
 }
 
+function ping_fips
+{
+    local fips=$(openstack ip floating list -f value -c IP)
+    for fip in $fips; do
+        ping -c 1 -w 60 $fip || complain 120 "cannot reach test VM at $fip."
```

Comment:
Yes, that was my point. We'll need some timeout when testing the network access during the upgrade, but before and after the upgrade it just should work (or there's something wrong). Anyway, it's a minor note.